src/net/unix/network.c

```diff
@@ -27,6 +31,8 @@
 /*------------------ Interfaces and sockets ------------------*/
 char *_zn_select_scout_iface()
 {
+    char *result;
```

Comment:
Pointer is not initialised but should be initialized to NULL.

Reply:
True. Thanks for pointing this out.

Reply:
Fixed.